apps/api/src/intern/intern.service.ts

```diff
@@ -213,37 +213,36 @@ export class InternService {
     Hvala na prijavi na DUMP Internship 2024. Uskoro ćemo te obavijestiti o sljedećim koracima prijave.
     Ako imaš pitanja oko internshipa ili procesa prijave slobodno nam se javi na info@dump.hr
     
-    U svakom trenutku možeš provjeriti status svoje prijave na https://internship.dump.hr/status/${newIntern.id}
-    
-    Lijep pozdrav,
-    
-    DUMP Udruga mladih programera
-    dump.hr`;
+    U svakom trenutku možeš provjeriti status svoje prijave na https://internship.dump.hr/status/${newIntern.id}`;
 
-    const marketingFormTextBody = `Pozdrav ${internToCreate.firstName},
+    const marketingFormAdditionalText = `U nastavku se nalazi link na formu, za područje Marketing, koju je obavezno ispuniti prije samog intervjua za DUMP Internship. 
+    Za ispunjavanje je predviđeno 15ak minuta, ali uzmi vremena koliko god ti treba. 
 
-    Hvala na prijavi na DUMP Internship 2024. Uskoro ćemo te obavijestiti o sljedećim koracima prijave.
-    Ako imaš pitanja oko internshipa ili procesa prijave slobodno nam se javi na info@dump.hr
-    
-    U svakom trenutku možeš provjeriti status svoje prijave na https://internship.dump.hr/status/${newIntern.id}
+    Link: https://forms.gle/EQxe44Kv9KFrryYf8`;
 
-    U nastavku se nalazi link na formu, za područje Marketing, koju je obavezno ispuniti prije samog intervjua za DUMP Internship. 
-    Za ispunjavanje je predviđeno 15ak minuta, ali uzmi vremena koliko god ti treba. 
+    const devFormAdditionalText = `U nastavku se nalazi link na primjer prošlogodišnjeg ispita za smjer programiranja na DUMP Internshipu.
+    Zadatke možeš rješavati u jednom od sljedećih jezika: JavaScript, Python, C#, C++, C, Java, Go, a za rješavanje je predviđeno 90 minuta.
 
-    Link: https://forms.gle/EQxe44Kv9KFrryYf8
-    
-    Lijep pozdrav,
+    Link: https://docs.google.com/document/d/1sxCjpDm58XdYDQzkDNK-6geArrR0C5EeKQmvW3F3Ou8/edit?usp=sharing`;
```

Comment:
Can you shorten this link to look better? Maybe use bitly for both the dev and marketing links but that is just a suggestion